docs/docusaurus.config.js

```diff
@@ -105,12 +105,12 @@ module.exports = {
           homePageId: "intro",
           sidebarPath: require.resolve("./sidebars.js"),
           // Please change this to your repo.
-          editUrl: "https://githlab.com/zmkproject/zmk/edit/main/docs/",
+          editUrl: "https://github.com/zmkfirmware/zmk/edit/main/docs/",
         },
         blog: {
           showReadingTime: true,
           // Please change this to your repo.
-          editUrl: "https://gitlab.com/zmkproject/zmk/edit/main/docs/blog/",
+          editUrl: "https://github.com/zmkfirmware/zmk/edit/main/docs/",
```

Comment:
This needs to be blog though, not docs.

Reply:
When I try this, the generated URLs are wrong. Blog posts link me to: https://github.com/zmkfirmware/zmk/edit/main/docs/blog/blog/2020-05-24-wip.md

Reply:
Oh! This is what I get for reviewing so early in the AM. Nevermind!